app/src/components/Movie/Movie.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Movie.scss';
+
+export default class Movie extends React.Component {
+  constructor() {
+    super();
+    this.selectThis = this.selectThis.bind(this);
+  }
+
+  selectThis(e) {
+    e.preventDefault();
+    this.props.selectMovie(this.props.item);
+  }
+
+  renderSelected() {
+    const item = this.props.item;
+    return (
+      <main className={styles.hero}>
+        <img src={item.poster} alt={item.show_title} />
+        <section>
+          <h2>{item.show_title}</h2>
+          <span className={styles.hero_rate}>{item.rating}</span>
+          <p>{item.category}</p>
+          <p>
+            <span className={styles.hero_time}>{item.release_year}</span>
+            <span className={styles.hero_time}>{item.runtime}</span>
+          </p>
+          <p className={styles.hero_summary}>{item.summary}</p>
+          <p className={styles.hero_crew}>
+            Director: {item.director}
+          </p>
+          <p className={styles.hero_crew}>
+            Cast: {item.show_cast}
+          </p>
+        </section>
+      </main>
+    );
+  }
+
+  renderListed() {
+    const item = this.props.item;
```

Comment:
destructuring to get show_title and so on?

Reply:
Done